airdrops/index_v3.json

```diff
@@ -340,6 +340,21 @@
                 "chain_id": 8453,
                 "decimals": 18
             }
+        },
+        "debridge_1": {
+            "file_path": "airdrops/debridge_1.parquet",
+            "file_hash": "aea1b729dea8ae1e2b412c5c6d821854cde4dc84ad996e54ba76439e08fabb58",
+            "asset_identifier": "DBR",
+            "url": "https://debridge.foundation/",
+            "name": "DBR",
+            "icon": "debridge.svg",
+            "icon_path": "airdrops/icons/debridge.svg",
+            "new_asset_data": {
+                "asset_type": "OWN_CHAIN",
```

Comment:
it seems to be a solana token
![image](https://github.com/user-attachments/assets/8d11496d-4df2-4ee4-945f-84f27b1c6a09)